src/math.cpp

```diff
@@ -0,0 +1,22 @@
+#include <cmath>
+#include <iostream>
+#include <stdexcept>
+
+#include "MiniLua/math.hpp"
+#include "MiniLua/stdlib.hpp"
+#include "MiniLua/utils.hpp"
+
+namespace minilua::math {
+
+auto abs(const CallContext& ctx) -> Value {
+    auto x = ctx.arguments().get(0);
+    auto res = minilua::to_number(ctx);
```

Comment:
Maybe only forward the arguments that are needed (unless this is specified by lua docs, but then add a comment why you do it that way).

Reply:
on second thought: should abs even be callable with string?

Reply:
when i tested it in lua, it worked like when l called it with an Number when the string is formated like a number. so i implemented it so that this works in our interpreter the same

Reply:
hmm ok then the lua documentation is incomplete. Can you add some documentation on the function for this?